system-int-test/src/cmr/system_int_test/data2/core.clj

```diff
@@ -319,7 +319,7 @@
 
 (defmethod item->metadata-result true
   [_ format-key item]
-  (let [{:keys [concept-id revision-id collection-concept-id]} item
+  (let [{:keys [concept-id _revision-id collection-concept-id]} item
```

Comment:
shouldn't we delete these kind of names when they are not used?

Reply:
Yes, they should be removed.